CHANGELOG.md

```diff
@@ -122,14 +133,30 @@
 * [ENHANCEMENT] Documented how to configure HA deduplication using Consul in a Mimir Helm deployment. #2972
 * [ENHANCEMENT] Improve `MimirQuerierAutoscalerNotActive` runbook. #3186
 * [ENHANCEMENT] Improve `MimirSchedulerQueriesStuck` runbook to reflect debug steps with querier auto-scaling enabled. #3223
+* [ENHANCEMENT] Mimir 2.4 release notes. #3104 #3319
+* [ENHANCEMENT] Use imperative for docs titles. #3178 #3332 #3343
+* [ENHANCEMENT] Docs: mention gRPC compression in "Production tips". #3201
+* [ENHANCEMENT] Update ADOPTERS.md. #3224 #3225
+* [ENHANCEMENT] Add a note for jsonnet deploying. #3213
+* [ENHANCEMENT] out-of-order runbook update with use case. #3253
+* [ENHANCEMENT] Fixed TSDB retention mentioned in the "Recover source blocks from ingesters" runbook. #3280
+* [ENHANCEMENT] Run Grafana Mimir in production using the Helm chart. #3072
+* [ENHANCEMENT] Use common config in the tutorial again. #3282
+* [ENHANCEMENT] Clarify changelog and remove duplicate flag in the documentation. #3370
+* [ENHANCEMENT] Updated detailed steps for migrating blocks from Thanos to Mimir. #3290
+* [ENHANCEMENT] Add scheme to DNS service discovery docs. #3450
+* [BUGFIX] Remove reference to file that no longer exists in contributing guide. #3404
+* [BUGFIX] Fix some minor typos in the contributing guide and on the runbooks page. #3418
+* [BUGFIX] Fix small typos in API reference. #3526
 * [BUGFIX] Fixed TSDB retention mentioned in the "Recover source blocks from ingesters" runbook. #3278
-* [BUGFIX] Fixed configuration example in the "Configuring the Grafana Mimir query-frontend to work with Prometheus" guide. #3373
+* [BUGFIX] Fixed configuration example in the "Configuring the Grafana Mimir query-frontend to work with Prometheus" guide. #3374
 
 ### Tools
 
-* [FEATURE] Add `copyblocks` tool, to copy Mimir blocks between two GCS buckets. #3263
+* [FEATURE] Add `copyblocks` tool, to copy Mimir blocks between two GCS buckets. #3264
 * [ENHANCEMENT] copyblocks: copy no-compact global markers and optimize min time filter check. #3268
 * [ENHANCEMENT] Mimir rules GitHub action: Added the ability to change default value of `label` when running `prepare` command. #3236
+* [ENHANCEMENT] Use delve compatible with Go 1.19.2. #3166
```

Comment:
This isn't user-facing. It only updated docker-compose dev setup under `./development/`.

Reply:
Done: 45b55e81d